rmgpy/cantherm/kinetics.py

```diff
@@ -144,9 +144,11 @@ def generateKinetics(self,Tlist=None):
             tunneling.E0_TS = (self.reaction.transitionState.conformer.E0.value_si*0.001,"kJ/mol")
             tunneling.E0_prod = (sum([product.conformer.E0.value_si for product in self.reaction.products])*0.001,"kJ/mol")
         elif tunneling is not None:
-            raise ValueError('Unknown tunneling model {0!r}.'.format(tunneling))
-        
-        logging.info('Generating {0} kinetics model for {0}...'.format(kineticsClass, self.reaction))
```

Comment:
Instead of deleting this logging statement, should we change it to a logging.debug statement?

Reply:
good idea